consensus/src/simplex/actors/resolver/ingress.rs

```diff
@@ -10,10 +10,6 @@ use futures::{channel::mpsc, SinkExt};
 use tracing::error;
 
 pub enum Message<S: Scheme, D: Digest> {
-    Fetch {
```

Comment:
We no longer need to explicitly fetch items from the resolver! We are just "informed"!